### PR TITLE
CXX-750 - Rename database::command to database::run_command

### PR DESCRIPTION
**Pull request by tessavitabile**

### CODE REVIEW

File: src/mongocxx/test/database.cpp [hunk 1]
```diff
@@ -225,6 +226,33 @@ TEST_CASE("A database", "[database]") {
...
+        });
+
+        database database = mongo_client[database_name];
+        bsoncxx::document::value command = bsoncxx::builder::stream::document{} 
+            << "command" << 1 << bsoncxx::builder::stream::finalize;
+        auto response = database.run_command(command);
+        REQUIRE(called);
+        REQUIRE(response.view()["foo"].get_int32() == 5);
+    }
+
```
Comment (acmorrow) [inline src/mongocxx/test/database.cpp]: +1 for tests

**Comment by acmorrow** [comment]
LGTM

File: src/mongocxx/test/database.cpp [hunk 2]
```diff
@@ -225,6 +226,33 @@ TEST_CASE("A database", "[database]") {
         collection obtained_collection = database[collection_name];
         REQUIRE(obtained_collection.name() == collection_name);
     }
+
+    SECTION("supports run_command") {
+        bool called = false;
+
+        bsoncxx::document::value doc = bsoncxx::builder::stream::document{} 
+            << "foo" << 5 << bsoncxx::builder::stream::finalize;
+        libbson::scoped_bson_t bsonDoc{doc};
```
Comment (samantharitter) [inline src/mongocxx/test/database.cpp]: nit, use snake case instead of camel, `bson_doc{doc}`

**Comment by samantharitter** [comment]
Just a style nit but LGTM!

File: src/mongocxx/test/database.cpp (hunk 2 above)
Comment (acmorrow) [inline src/mongocxx/test/database.cpp]: +1, good catch, thanks.

**Comment by amidvidy** [comment]
```
  _      _____ _______ __  __ 
 | |    / ____|__   __|  \/  |
 | |   | |  __   | |  | \  / |
 | |   | | |_ |  | |  | |\/| |
 | |___| |__| |  | |  | |  | |
 |______\_____|  |_|  |_|  |_|
```

**Pull request merged**